packages/user/src/components/Invitation/InvitationFormFields.tsx

```diff
@@ -11,21 +11,24 @@ import { Button } from "primereact/button";
 import React, { useEffect, useState } from "react";
 
 import { InvitationAppOption, InvitationRoleOption } from "@/types";
+import { RenderAdditionalInvitationFields } from "@/types/invitation";
 
 interface IProperties {
+  additionalFields?: RenderAdditionalInvitationFields;
```

Comment:
Lets rename it to `renderAdditionalFields`. This would give more sense of what the prop is about.